central/complianceoperator/v2/scanconfigurations/datastore/datastore_impl_test.go

```diff
@@ -217,6 +217,41 @@ func (s *complianceScanConfigDataStoreTestSuite) TestGetScanConfiguration() {
 	}
 }
 
+func (s *complianceScanConfigDataStoreTestSuite) TestRemoveClusterFromScanConfig() {
+	configID1 := uuid.NewV4().String()
+	scanConfig1 := s.getTestRec(mockScanName)
+	scanConfig1.Id = configID1
+	configID2 := uuid.NewV4().String()
+	scanConfig2 := s.getTestRec("mock-scan-config-2")
+	scanConfig2.Id = configID2
+	// Add a record so we have something to find
+	s.Require().NoError(s.storage.Upsert(s.testContexts[unrestrictedReadWriteCtx], scanConfig1))
+	s.Require().NoError(s.storage.Upsert(s.testContexts[unrestrictedReadWriteCtx], scanConfig2))
+
+	for _, cluster := range scanConfig1.Clusters {
+		s.Require().NoError(s.dataStore.UpdateClusterStatus(s.testContexts[unrestrictedReadWriteCtx], configID1, cluster.ClusterId, "testing status"))
+
+	}
+
+	for _, cluster := range scanConfig2.Clusters {
+		s.Require().NoError(s.dataStore.UpdateClusterStatus(s.testContexts[unrestrictedReadWriteCtx], configID2, cluster.ClusterId, "testing status"))
+	}
+
+	err := s.dataStore.RemoveClusterFromScanConfig(s.testContexts[unrestrictedReadWriteCtx], scanConfig1.Clusters[0].GetClusterId())
+	s.Require().NoError(err)
+	newscanConfig, exists, err := s.dataStore.GetScanConfiguration(s.testContexts[unrestrictedReadWriteCtx], scanConfig1.GetId())
```

Comment:
nit:  I would add a new line after each block of checks.  Makes it easier to read.